lib/eventDrops.js

```diff
@@ -139,11 +139,11 @@ module.exports = function (d3) {
         zoom.x(xScale);
 
         function updateZoom() {
-          if (d3.event.sourceEvent.toString() === '[object MouseEvent]') {
+          if (!d3.event.sourceEvent && d3.event.sourceEvent.toString() === '[object MouseEvent]') {
```

Comment:
Shouldn't you remove the `!`? Looks weird to check that the object is null to test then if its string representation is an object.